### PR TITLE
Added an omnipresent haiku

### PR DESCRIPTION
**Pull request by rajatshenoy56**

### CODE REVIEW

File: _haikus/omnipresent.md
Comment (ltagliaferri) [inline _haikus/omnipresent.md:9]: This last line is one syllable too long. You can try:

```
child," Cloud replied.<br>
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for submitting this haiku! There are some changes that need to be made to the syllable count in order to have a valid haiku. The test suite on our haiku will let you know which lines are failing, please check the **details** under "All checks have failed" of this pull request to figure out which lines need to be modified.

A haiku is composed of 3 lines. The first line is 5 syllables long, the second line is 7 syllables long, and the third line is 5 syllables long. Although the majority of the haikus here are written in English, you can submit a haiku in any human language.

Please consult our [contributing guidelines](https://github.com/do-community/cloud_haiku#contributing) if you need extra guidance. Once you amend this pull request, we can merge it in.

*Please note: after inactivity, pull requests are subject to being closed and marked invalid by maintainers.*

**Comment by rajatshenoy56** [comment]
The last line of the haiku was updated

**Review by ltagliaferri — APPROVED** [review]
Awesome, thank you for the update and for contributing to open source 🎉

**Comment by rajatshenoy56** [comment]
Thank you so much!

**Pull request merged**
